dldashboard/web/src/StateManager.js

```diff
@@ -238,7 +289,7 @@ class StateManager {
           depth: depth,
         });
       } else if (ref instanceof LiveObjects || ref instanceof LiveHumans) {
-        if (res.object_image !== -1) {
+        if (res.object_image !== -1 && res.object_image !== undefined) {
```

Comment:
I thought we won't be setting the state of LiveObjects and LiveImage here since we are handling them in a separate event above